tests/test_ads.py

```diff
@@ -561,6 +561,61 @@ def test_bytes_from_dict(self):
         with self.assertRaises(KeyError):
             pyads.bytes_from_dict(OrderedDict(), structure_def)
 
+    def test_dict_slice_generator(self):
```

Comment:
Can we remove the test slice generators as they are not used?

Reply:
do you mean just remove the tests? as the functons are still used in the read/write_list_by name functions in ads.py.

Reply:
even though the tests for the read/write_list `test_read_list_ads_sub_commands` technically cover the usage of the _slice_generators, I would argue as they are standalone functions, it is still important to test them with tests spefic to test those standalone functions?

Reply:
Ah sorry I didn't see that these are tests. I thought it to be a generator for some test data so I looked if they are called anywhere in the code which was not the case. Alright then all is set and I shall merge this PR :)